### ISSUE TITLE
Feature Request.

### ISSUE THREAD

**Issue by justjoolz** [pixelandtonic/CategorySources#1]
Could you use the same technique here to display the actual Categories data like the Structures are in the Entries Section?

And then also hide the whole Categories section from the top menu.

I have a client who is confused by the term 'categories' but wants their functionality ie. the parent level structure entries being selected automatically.

**Comment by brandonkelly** [comment]
No, you can't mix element types on the same index page, at least without completely recoding the index page.

**Comment by justjoolz** [comment]
How about just having anchor links next to the titles ?
ie. to /admin/categories/programs/15-spiritual
Would that be easy to implement ?
